importexport/inc/class.importexport_import_ui.inc.php

```diff
@@ -74,7 +74,9 @@ public function import_dialog($content = array()) {
 					$plugin = new $definition_obj->plugin;
 
 					// Check file encoding matches import
-					$sample = file_get_contents($content['file']['tmp_name'],false, null, 0, 1024);
+					$sample = file_get_contents($content['file']['tmp_name']);
+					$sample = mb_substr( $sample, 1024 );
+
```

Comment:
I'd rather use the following:
```
$sample = mb_substr(file_get_contents($content['file']['tmp_name'], false, null, 0, 2048), 1024);
```
Please do not assign to $sample twice, as this is flagged as potential mistake in Netbeans.